addon/services/items-service.js

```diff
@@ -15,77 +26,101 @@ export default Service.extend(serviceMixin, {
   /**
    * Item Search
    */
-  search (form, portalOpts) {
-    const qs = this.encodeForm(form);
-    const urlPath = `/search?${qs}&f=json`;
-    return this.request(urlPath, null, portalOpts);
+  search (form/* ,portalOpts */) {
+    const session = this.get('session.authMgr');
```

Comment:
suggest naming this `const authentication` for clarity and ease of passing to rest-js fns